python_todo/views.py

```diff
@@ -0,0 +1,44 @@
+from flask import Flask, jsonify, request, render_template
+from python_todo import app, db
+from python_todo.models import Task
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+@app.route('/task', methods=['GET'])
```

Comment:
Typically, rest resource names are plural. So, you'd have `/tasks` and `/tasks/:id`.